COORDINATION.md

```diff
@@ -64,6 +64,14 @@ The NVMe device driver will poll the CSTS.rdy bit to ensure that infrastructure
 
 ## 3: Out-band via platform BMC
 
+### OOB proposal with PCI architected Extended Capability Structure in the xPU
+
+OOB refers to USB2 access to the xPU from the server BMC.  USB2 implies future products, and is necessary as a high speed OOB interface for supporting large files associated with FW/SW upgrade and audit logs.
```

Comment:
There needs to be a general section before this on that discribes OOB in general.  This would go better with the USB section below.  The diagram section below this needs to go first because it is supposed to represent OOB generally without the physical or protocol specifics.

I believe the PCIe reference here is to changes in the CEM spec.  Please refer to those specs so it is clear what we are talking about.